vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story

```diff
@@ -285,6 +285,67 @@ When I scan barcode from screen and save result to scenario variable `qrCodeLink
 Then `${qrCodeLink}` is = `https://github.com/vividus-framework/vividus`
 
 
+Scenario: Verify steps: "When I set value of Android slider located `$locator` to `$number`"
+Meta:
+    @targetPlatform android
+When I tap on element located `accessibilityId(menuToggler)`
+When I tap on element located `xpath(<menuSliderXpath>)`
+When I wait until element located `accessibilityId(zeroToHundredSlider)` appears
+When I set value of Android slider located `accessibilityId(zeroToHundredSlider)` to `77.0`
+When I save `${value-attribute}` attribute value of element located `accessibilityId(zeroToHundredSliderPosition)` to scenario variable `sliderState`
+Then `${sliderState}` is equal to `77.0`
+When I set value of Android slider located `accessibilityId(zeroToHundredSlider)` to `13.0`
+When I save `${value-attribute}` attribute value of element located `accessibilityId(zeroToHundredSliderPosition)` to scenario variable `sliderState`
+Then `${sliderState}` is equal to `13.0`
+When I set value of Android slider located `accessibilityId(negativeFiftyToFiftySlider)` to `25.0`
+When I save `${value-attribute}` attribute value of element located `accessibilityId(negativeFiftyToFiftySliderPosition)` to scenario variable `sliderState`
+Then `${sliderState}` is equal to `-25.0`
+When I set value of Android slider located `accessibilityId(negativeFiftyToFiftySlider)` to `70.0`
+When I save `${value-attribute}` attribute value of element located `accessibilityId(negativeFiftyToFiftySliderPosition)` to scenario variable `sliderState`
+Then `${sliderState}` is equal to `20.0`
+When I set value of Android slider located `accessibilityId(eighteenToSixtyFiveSlider)` to `47.0`
+When I save `${value-attribute}` attribute value of element located `accessibilityId(eighteenToSixtyFiveSliderPosition)` to scenario variable `sliderState`
+Then `${sliderState}` is equal to `65.0`
+When I set value of Android slider located `accessibilityId(eighteenToSixtyFiveSlider)` to `20.0`
+When I save `${value-attribute}` attribute value of element located `accessibilityId(eighteenToSixtyFiveSliderPosition)` to scenario variable `sliderState`
+Then `${sliderState}` is equal to `38.0`
+When I set value of Android slider located `accessibilityId(eighteenToSixtyFiveSlider)` to `2.0`
+When I save `${value-attribute}` attribute value of element located `accessibilityId(eighteenToSixtyFiveSliderPosition)` to scenario variable `sliderState`
+Then `${sliderState}` is equal to `20.0`
```

Comment:
what about moving repeating steps to scenario with ExamplesTable?

Reply:
Wanted to make scenarios more less generic, so if we decide to move the into some TMS there won't be a question how we get this particular screen, actually I wanted to move this logic into composites, what do you think?